flang/test/Lower/loops.f90

```diff
@@ -84,5 +85,19 @@ subroutine loop_test
   print '(" F:",X,F3.1,A,I2)', x, ' -', xsum
 end subroutine loop_test
 
+! CHECK-LABEL: print_nothing
+subroutine print_nothing(k1, k2)
+  if (k1 > 0) then
+    ! CHECK: br [[header:\^bb2]]
+    ! CHECK: [[header]]
```

Comment:
Should this be checking for the preheader here, since that seems to be the regression?

Reply:
Yes, that is what is being checked.  This is the branch in the preheader block to header block `header`.